quickstarts/uppercase-firestore/README.md

```diff
@@ -1,19 +1,64 @@
 # Firebase SDK for Cloud Functions Quickstart - Cloud Firestore
 
-This quickstart demonstrates using **Firebase SDK for Cloud Functions** setup
-with a **Cloud Firestore**.
+This quickstart demonstrates using the **Firebase SDK for Cloud Functions** with
+**Cloud Firestore**.
 
 ## Introduction
 
 This sample app does two things:
 
-- Create messages in Cloud Firestore using a simple HTTPS request which is
+- Creates messages in Cloud Firestore using a simple HTTPS request which is
   handled by an HTTP function. Writing to Cloud Firestore is done using the
   Firebase Admin SDK.
 - When a message gets added in the Cloud Firestore, a function triggers and
   automatically makes these messages all uppercase.
 
-## Deploy and test
+## Run locally with the Firebase Emulator suite
+
+The
+[Firebase Local Emulator Suite](https://firebase.google.com/docs/emulator-suite)
+allows you to build and test apps on your local machine instead of deploying to
+a Firebase project.
+
+1. Create a Firebase project in the
+   [Firebase Console](https://console.firebase.google.com)
+   > _Wondering why this step is needed?_ Even though the emulator will run this
+   > sample on your local machine, it needs to associate with a Firebase project
+   > to retrieve some configuration values.
+1. [Set up or update the Firebase CLI](https://firebase.google.com/docs/cli#setup_update_cli)
+1. Run `firebase emulators:start`
+1. Open the Emulator Suite UI
+   1. Look in the output of the `firebase emulators:start` command for the url
```

Comment:
Should URL be capped?  If it's a literal, use backticks.

Reply:
Switched to capitalized URL